### PR TITLE
Update presentation_layout@es.md

### PR DESCRIPTION
**Pull request by SidVal**
404's fixed
https://www.ampproject.org/es/docs/guides/responsive/style_pages
https://www.ampproject.org/es/docs/guides/author-develop/responsive/style_pages (error)

https://www.ampproject.org/es/docs/guides/responsive/control_layout
https://www.ampproject.org/es/docs/guides/author-develop/responsive/control_layout (error)

### CODE REVIEW

File: content/docs/tutorials/create/presentation_layout@es.md [hunk 1]
```diff
@@ -19,12 +19,12 @@ Las AMP son páginas web; el diseño de estas y de sus elementos se realiza a tr
 </style>
 [/sourcecode]
 
-Las páginas AMP pueden tener solo una hoja de estilo integrada y hay ciertos selectores que no puedes usar. [Aprende todo acerca del estilo](/es/docs/guides/author-develop/responsive/style_pages.html).
+Las páginas AMP pueden tener solo una hoja de estilo integrada y hay ciertos selectores que no puedes usar. [Aprende todo acerca del estilo](/es/docs/guides/responsive/style_pages).
```
Comment (bpaduch) [inline content/docs/tutorials/create/presentation_layout@es.md]: Please add .html to the link. Thank you.

File: content/docs/tutorials/create/presentation_layout@es.md [hunk 2]
```diff
@@ -19,12 +19,12 @@ Las AMP son páginas web; el diseño de estas y de sus elementos se realiza a tr
...
 
 ## Controla el diseño
 
 AMP sigue reglas estrictas al distribuir elementos en la página. En una página HTML normal, CSS se usa casi exclusivamente para distribuir elementos. Sin embargo, por motivos de rendimiento, AMP requiere que todos los elementos tengan un tamaño explícito configurado desde el principio.
 
-Aprende todo acerca de cómo AMP representa y diseña una página y cómo puedes modificar el diseño en [Cómo controlar el diseño](/es/docs/guides/author-develop/responsive/control_layout.html).
+Aprende todo acerca de cómo AMP representa y diseña una página y cómo puedes modificar el diseño en [Cómo controlar el diseño](/es/docs/guides/responsive/control_layout).
```
Comment (bpaduch) [inline content/docs/tutorials/create/presentation_layout@es.md]: Please add .html to the link. Thank you.

**Comment by SidVal** [comment]
Done :+1:

**Comment by bpaduch** [comment]
Will merge when build is fixed.

**Pull request merged**